.github/codemention.yml

```diff
@@ -0,0 +1,23 @@
+rules:
+  - patterns: ['**/*']
+    mentions: ['szdziedzic', 'khamilowicz', 'sjchmiela']
+  - patterns: ['packages/eas-cli/src/build/**']
+    mentions: ['szdziedzic', 'khamilowicz', 'sjchmiela']
+  - patterns: ['packages/eas-cli/src/commands/build/**']
+    mentions: ['szdziedzic', 'khamilowicz', 'sjchmiela']
+  - patterns: ['packages/eas-cli/src/submit/**']
+    mentions: ['szdziedzic', 'khamilowicz', 'sjchmiela']
+  - patterns: ['packages/eas-cli/src/commands/submit.ts']
+    mentions: ['szdziedzic', 'khamilowicz', 'sjchmiela']
```

Comment:
can we add @radoslawkrzemien here as well? 🙏 although I can do it in separate PR to not block you